feat(popover-core):  prevent internal logic (#839)
scania-digital-design-system/tegel: packages/core/src/components/popover-core/readme.md

# tds-core-popover

## Controlled vs Uncontrolled

> - **Controlled**: 
>   How does component enter controlled mode?
>   - Leaving `selector` or `referenceEl` prop empty
> 

Reviewer: ```suggestion
- **Controlled**: 
  How does component enter controlled mode?
  - Setting the `show` prop to either `true` or `false`
```
